Week3/homework/step2-2.js

```diff
@@ -1,20 +1,28 @@
 'use strict';
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
-  const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  // Array from with map method
+  const numbers = Array.from(Array(stopIndex - startIndex + 1)).map(
```

Comment:
Impressive solution! 👍

Reply:
Thank you. I have just trying to do build an arr without for loops.